docs/_embed/partner_api.md

```diff
@@ -140,11 +140,15 @@ The idea is to use an intermediate page that reads the access token from the URL
 
 Available parameters to the Apps resource:
 
-| parameter     | requirement | notes                                                                                  |
-| ------------- | ----------- | -------------------------------------------------------------------------------------- |
-| **client_id** | Required    | Your application client ID.                                                            |
-| **per_page**  | Optional    | (defaults to 100, max of 100) Limit the number of apps returned.                       |
-| **page**      | Optional    | (defaults to 1) The page number. Page number 1 refers to the first page in the result. |
+| parameter                   | requirement | notes                                                                                         |
+| --------------------------- | ----------- | --------------------------------------------------------------------------------------------- |
+| **client_id**               | Required    | Your application client ID.                                                                   |
+| **category**                | Optional    | Filter the results by app category.                                                           |
+| **is_in_zap_template_with** | Optional    | Fetch apps that appear in a Zap Template with your app. |
+| **title_search**            | Optional    | Filter the results by matching title (case-insensitive).                                      |
```

Comment:
Users can only provide one title there a curl example below but maybe it's not clear? Couple options would be to. Mention it in parens like we do for other parameters like `Filter the results by matching title (case-insensitive single title)`. Alternatively add an extra column called "type" would provide better specificity.
So for the `title_search` case the type would be `string`.

Reply:
To be clear, `title_search` will search for the string anywhere in the title, not just exact matches. So it can return multiple results - you can search for the letter "s" if you like!

But you have to do multiple searches to search for multiple different strings.

Reply:
Ok this sounds like it would be more accurate:
`Filter the results by performing a case-insensitive search for a substring within the app title.`

Reply:
Thanks for the clarification! Could use following for a shorter clause: Filter the results with a case-insensitive search for a substring within the app title.